docs/00_getting_started_sd.md

```diff
@@ -61,37 +62,36 @@ SD:\
 ├── (a rom).n64
 ├── (some folder with roms)\
         │   └── (some folder with roms)\
-        |       └── (Some supported ROM files)
+        |       └── (some supported ROM files)
         │
-        ├── (Some supported ROM files)
+        ├── (some supported ROM files)
         |
-        └── (Some folder with 64DD disk images)\
-            └── (Some 64DD disk images)
+        └── (some folder with 64DD disk images)\
+            └── (some 64DD disk images)
 ```
 
 
 ## Save Files
-All save files (whether `FlashRam`, `SRAM` or `EEPROM`) use the `.sav` extension and match the filename of a ROM.
-
-Each save file can be found in the `/saves` folder located in the **same directory** as the ROM and shares the same file name, apart from the extension.
-These files are created and modified when a "game" saves.
+All save files (whether `FlashRam`, `SRAM` or `EEPROM`) will be read from a `/saves` folder located **in the same directory as the ROM** 
+and they must share the same file name, but use the `.sav` extension. `.sav` files will be created and modified whenever a ROM file saves to 
+the "cartridge save memory".
 
 ```plaintext
 ├── (some folder with roms)\
     ├── a_rom.z64
-    ├── b_rom.n64
+    ├── b_rom_whatevs.n64
     └── saves\
         ├── a_rom.sav
-        └── b_rom.sav
+        └── b_rom_whatevs.sav
```

Comment:
Not a fan of `whatevs`

Reply:
Fine by me, the intent here was to have two different ROM names, so the point is more clear.

Reply:
It is more the language... `whatevs` is slang which some people will not understand.

Reply:
I tried to be funny and I failed. Changed to something more serious.